sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -871,18 +854,18 @@ namespace Azure { namespace Storage { namespace DataLake {
     public:
       struct CreateOptions
       {
-        std::string ClientRequestId; // Provides a client-generated, opaque value with a 1 KB
-                                     // character limit that is recorded in the analytics logs when
-                                     // storage analytics logging is enabled.
-        int32_t Timeout
-            = int32_t(); // The timeout parameter is expressed in seconds. For more
-                         // information, see <a
-                         // href="https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations">Setting
-                         // Timeouts for Blob Service Operations.</a>
+        Azure::Core::Nullable<std::string>
+            ClientRequestId; // Provides a client-generated, opaque value with a 1 KB character
+                             // limit that is recorded in the analytics logs when storage analytics
+                             // logging is enabled.
+        Azure::Core::Nullable<int32_t>
```

Comment:
Done via context

Reply:
This is not exposed to customers as this is protocol layer options.